drawers/graphic_field.go

```diff
@@ -0,0 +1,46 @@
+package drawers
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+
+	"github.com/fogleman/gg"
+	"github.com/ingridhq/zebrash/elements"
+)
+
+func NewGraphicFieldDrawer() *ElementDrawer {
+	return &ElementDrawer{
+		Draw: func(gCtx *gg.Context, element interface{}, _ DrawerOptions) error {
+			field, ok := element.(*elements.GraphicField)
+			if !ok {
+				return nil
+			}
+
+			if field.DataBytes != len(field.Data) {
+				return fmt.Errorf("unexpected length of graphic field data, expected %d bytes", field.DataBytes)
+			}
+
+			width := field.RowBytes * 8
+			height := field.DataBytes / field.RowBytes
+
+			img := image.NewGray(image.Rect(0, 0, width, height))
+
+			for y := 0; y < height; y++ {
+				for x := 0; x < width; x++ {
+					val := ((field.Data[y*(width/8)+x/8]) >> (7 - x%8)) & 0xF
```

Comment:
This line is hard to understand, could you move it to a function or at least add a comment?

Reply:
@mknycha-ingrid Added comment, please check if it is better to understand this way